pyaedt/modules/PostProcessor.py

```diff
@@ -4614,7 +4589,7 @@ def create_ami_statistical_eye_plot(
         ami_plot_type : str, optional
             String containing the report AMI type. Default is ``"InitialEye"``. It can be ``"EyeAfterSource"``,
             ``"EyeAfterChannel"`` or ``"EyeAfterProbe"``.
-        plotname : str, optional
+        plot_name : str, optional
             The name of the plot.  Defaults to a unique name starting with ``"Plot"``.
```

Comment:
```suggestion
            Plot name.  The default is ``None``, in which case
            a unique name starting with ``"Plot"`` is automatically assigned.
```
Is the prefix ``"Plot"`` also assigned to other occurrences of this parameter? If so, all descriptions should be made consistent and indicate this.